### PR TITLE
#CNV-30296 - new rn for DPDK docs

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.15 RN

Issue: [CNV-30296](https://issues.redhat.com//browse/CNV-30296)

Link to docs preview: https://71859--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-networking

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by tmalove** [comment]
@sabrinajess will you add a preview link? Thanks!

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -95,7 +95,7 @@ This release adds new features and enhancements related to the following compone
 //CNV-13680: Connecting OVN-Kubernetes to underlying physical networks. Remove known issue BZ-1885605 after adding this note.
 
 //CNV-30296: Running DPDK workloads on SR-IOV is now GA. The TP note from 4.13 should have been part of 4.14 as well.
-
+* xref:../../virt/vm_networking//virt-using-dpdk-with-sriov.adoc#virt-configuring-cluster-dpdk_virt-using-dpdk-with-sriov[Configuring a cluster for DPDK workloads on SR-IOV] was previously Technology Preview and is now generally available.
```
Comment (segevyos) [inline virt/release_notes/virt-4-15-release-notes.adoc:98]: 1. Shouldn't "generally available" be capitalized (i.e. "Generally Available"), just like "Technology Preview" that precedes it?
2. I'm not sure that "Configuring a cluster for DPDK workloads" is the interesting topic that is now GA'd. I think that the highlight which is now GA'd is [running the DPDK checkup](https://docs.openshift.com/container-platform/4.15/virt/monitoring/virt-running-cluster-checkups.html#virt-checking-cluster-dpdk-readiness_virt-running-cluster-checkups), and "Configuring a cluster for DPDK workloads" is a prerequisite for that.
@orelmisan @RamLavi @bell-lev Please keep me honest here. Thank you

**Review by segevyos — CHANGES_REQUESTED** [review]

File: virt/release_notes/virt-4-15-release-notes.adoc (hunk 1 above)
Comment (orelmisan) [reply virt/release_notes/virt-4-15-release-notes.adoc:98]: The DPDK checkup is still a tech preview.
DPDK support in OpenShift Virtualization is GA.

Comment (segevyos) [reply virt/release_notes/virt-4-15-release-notes.adoc:98]: Thank you Orel.
So this leaves us only with my first comment (about the capitalization).

Comment (segevyos) [reply virt/release_notes/virt-4-15-release-notes.adoc:98]: Acceptable.
/lgtm then

**Pull request merged**